docs/charging.md

```diff
@@ -24,43 +24,63 @@ end
 
 ## 2. Send the nonce to your backend
 
-Send the nonce created from `cardNonceResponseReceived` to your backeend.
+Send the nonce and buyer verification token created from `cardNonceResponseReceived` to your backend.
 
 ```
 class PaymentPage extends React.Component {
 
-  cardNonceResponseReceived = (errors, nonce, cardData) => {
+  cardNonceResponseReceived = (errors, nonce, cardData, buyerVerificationToken) => {
     ...
     alert("nonce created: " + nonce)
-    API.post('/payments', data: { nonce: nonce }) // implement this
+    API.post('/payments', data: { nonce: nonce, token: buyerVerificationToken }) // implement this
   }
 }
 ```
-## 3. Get your API access token
 
-To get your API access token:
+## 3. Get your Sandbox credentials
+
+To get your Sandbox credentials:
+
 1. Open your [Application Dashboard](https://connect.squareup.com/apps).
-2. Click on the application you want to use for the payment form.
-3. Click on the "Show" link on the **Sandbox Access Token** and copy it.
+1. Click on the application you want to use for the payment form. You should land on the "Credentials" tab.
+1. On the bottom left corner, toggle "Production Settings" to "Sandbox Settings" .
+1. Click on the "Show" link on the **Sandbox Access Token** and copy it.
+1. Click on the "Locations" tab on the left navigation bar.
+1. Copy the **Sandbox Location ID**.
 
 ## 4. Take a payment
 
-On your backend, take this nonce and create a charge using Square's Transactions API. You can read the [Transactions API Setup Guide](https://docs.connect.squareup.com/payments/transactions/setup) to learn how to create a charge.
+On your backend, take this nonce and create a charge using Square's Payments API. You can read the [Take and Refund Payments](https://developer.squareup.com/docs/payments-api/take-and-refund-payments) guide to learn how to create a payment.
 
 ```
 def create
   payload = {
-    "card_nonce": params[:nonce],
+    "source_id": params[:nonce],
+    "verification_token": params[:token],
+    "autocomplete": true,
+    "location_id": SANDBOX_LOCATION_ID,
     "amount_money": { # amount_money = $1.00
       "amount": 100,
       "currency" "USD"
     },
     "idempotency_key": SecureRandom.uuid
   }
-  url = "https://connect.squareupsandbox.com/v2/locations/{SANDBOX_LOCATION_ID}/transactions"
+  url = "https://connect.squareupsandbox.com/v2/payments"
   res = HTTP.auth("Bearer #{SANDBOX_ACCESS_TOKEN}").post(url, :body => payload.to_json)
   render json: res.body
 end
 ```
 
 The [idempotency key](https://docs.connect.squareup.com/basics/api101/idempotency) should be generated as a UUID by the backend payment page.
```

Comment:
This should also target https://developer.squareup.com/docs/working-with-apis/idempotency